x/sequencers/handler.go

```diff
@@ -0,0 +1,28 @@
+package sequencers
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/dymensionxyz/dymension-rdk/x/sequencers/keeper"
+	"github.com/dymensionxyz/dymension-rdk/x/sequencers/types"
+)
+
+// NewHandler returns a new msg handler.
+func NewHandler(k keeper.Keeper) sdk.Handler {
```

Comment:
I believe this file is redundant. in sdk 0.46 you simply need to simply register your msg server infront of your keeper and implement the corresponding methods in the keeper.